docs/oasis-network-primer/token_distribution/pie/Chart.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PlotlyChart from '@site/src/PlotlyChart';
+import FallbackSvg from '!!file-loader!./fallback.svg';
+
+import type { PlotlyDataLayoutConfig } from 'plotly.js-basic-dist';
+
+import data from '../monthly_data.json';
+
+const sum = (values: number[]) => {
+  return values.reduce((acc, value) => acc + value, 0);
+};
+
+const FinalDistributionPieChart = () => {
+  const chart: PlotlyDataLayoutConfig = {
+    data: [
+      {
+        values: [
+          sum(data.coreContributors),
+          sum(data.backers),
+          sum(data.foundation),
+          sum(data.communityAndEcosystem),
+          sum(data.strategicPartnersAndReserve),
+          2_350_000_000,
+        ],
+        labels: [
+          'Core Contributors',
```

Comment:
Move labels literals to `data.labels.coreContributors`, `data.labels.backers` etc. so that they won't be duplicated in `token_distribution/Chart.tsx`.